Automattic/jetpack
# Update remove creator card and handle growth bundle on applicable product classes

## Proposed changes:

* Remove Creator card from My Jetpack
* Update Social and Stats paid plan check to account for the growth bundle

### Other information:


## Jetpack product discussion

P2: pbNhbs-bxX-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch via the Jetpack Beta plugin with sandbox access enabled or your local dev environment
2. Sandbox your environment
3. Go to `/wp-admin/admin.php?page=my-jetpack#/` and confirm that you do not see a Creator card
![image](https://github.com/user-attachments/assets/168c149e-8d74-4d1c-9f12-610ff0f33e47)
4. Sandbox `public-api.wordpress.com`
5. In your 0-sandbox.php file, add the following line
```
lang=php
define( 'USE_STORE_SANDBOX', true );
```
6. Go to https://wordpress.com/checkout/jetpack/jetpack_growth_yearly and purchase the product
7. Go to `/wp-admin/admin.php?page=my-jetpack#/add-license` and activate the license
8. Go back to `/wp-admin/admin.php?page=my-jetpack#/` and ensure you see the growth product in the Your Plan section
![image](https://github.com/user-attachments/assets/925af0f2-8309-472c-b2e2-e56e77422e5f)
9. Make sure Stats and Social both do not have an `Upgrade` CTA
![image](https://github.com/user-attachments/assets/bb010cbb-752f-4314-b516-768abe1f3b89)

Bonus
10. Either purchase a Creator plan for your site or assign it via the Store Admin
11. Go back to My Jetpack and ensure you still see the Creator plan in the Your Plan section
![image](https://github.com/user-attachments/assets/9b02ea91-fdfa-4ae1-a7c9-e16a8ff9fb24)
